src/main/java/useCases/createCorporatePage/PageGenerateDsRequestModel.java

```diff
@@ -0,0 +1,13 @@
+package useCases.createCorporatePage;
+
+import entity.User;
+
+public class PageGenerateDsRequestModel {
+    private final String pageLabel;
+    private User owner;
+
+    public PageGenerateDsRequestModel(String pageLabel, User owner) {
```

Comment:
I'm assuming this is the data to be stored in the db. We might want more information stored than just pageLabel and owner. For example pageInfo.